docs/src/pages/guides/FHIRServerUsersGuide.md

```diff
@@ -1,9 +1,6 @@
 ---
 layout: post
 title:  IBM FHIR Server User's Guide
-description: IBM FHIR Server User's Guide
-Copyright: years 2017, 2022
```

Comment:
how come these lines are being removed?

Reply:
i found they're not actually used for anything and so I was trying to normalize our markdown files a bit (they all have slightly different metadata at the top).  i'm not sure it matters though...happy to back out if you think this info belongs up here